src/java/com/zimbra/graphql/resolvers/impl/AccountResolver.java

```diff
@@ -59,6 +60,16 @@ public AccountInfo accountInfoGet(@GraphQLRootContext RequestContext context)
         return accountRepository.accountInfoGet(context);
     }
 
+    @GraphQLQuery(description="Retrieve info")
```

Comment:
If possible a description that makes it clear how this differs from accountInfo would be helpful. Also rename `accountInfoGet` to `accountInfo`.